### PR TITLE
feat(azure): add info on how to use dockerized az CLI

### PR DESCRIPTION
**Pull request by coderbyheart**

### CODE REVIEW

File: docs/azure/GettingStarted/AzureCLI.rst [hunk 1]
```diff
@@ -12,3 +12,23 @@ After installing the CLI, you can execute the ``az`` command:
    # ...
    # azure-cli 2.24.0
    # ...
+
+Create the :file:`az` alias when using Docker
```
Comment (peknis) [inline docs/azure/GettingStarted/AzureCLI.rst]: I believe we should avoid any special formatting in headers.

File: docs/azure/GettingStarted/AzureCLI.rst [hunk 2]
```diff
@@ -12,3 +12,23 @@ After installing the CLI, you can execute the ``az`` command:
...
+*********************************************
+
+The installation instructions expect the :file:`az` command to be run from the current directory.
+Therfore an alias needs to be created, when using the dockerized Azure CLI.
```
Comment (peknis) [inline docs/azure/GettingStarted/AzureCLI.rst]:
```suggestion
The installation instructions expect that you run the ``az`` command from the current directory.
Therfore, you need to create an alias, when using the dockerized Azure CLI.
```

File: docs/azure/GettingStarted/AzureCLI.rst [hunk 3]
```diff
@@ -12,3 +12,23 @@ After installing the CLI, you can execute the ``az`` command:
...
+
+1. Copy the script to an executable file in your path, for example :file:`~/bin/az`:
+
+   .. literalinclude:: ./scripts/az
+      :language: bash
+
+#. Make sure that you can call it using :file:`az` (the folder :file:`~/bin` needs to be in your ``$PATH``).
+   This enables you to execute :file:`az` when using the dockerized Azure CLI.
+
+#. Log-in again, in order to continue with the next step:
```
Comment (peknis) [inline docs/azure/GettingStarted/AzureCLI.rst]:
```suggestion
#. Log in again to continue with the next step:
```

File: docs/azure/GettingStarted/AzureCLI.rst [hunk 4]
```diff
@@ -12,3 +12,23 @@ After installing the CLI, you can execute the ``az`` command:
...
+The installation instructions expect the :file:`az` command to be run from the current directory.
+Therfore an alias needs to be created, when using the dockerized Azure CLI.
+
+1. Copy the script to an executable file in your path, for example :file:`~/bin/az`:
+
+   .. literalinclude:: ./scripts/az
+      :language: bash
+
+#. Make sure that you can call it using :file:`az` (the folder :file:`~/bin` needs to be in your ``$PATH``).
+   This enables you to execute :file:`az` when using the dockerized Azure CLI.
```
Comment (peknis) [inline docs/azure/GettingStarted/AzureCLI.rst]:
```suggestion
#. Make sure that you can call it using ``az`` (the folder :file:`~/bin` needs to be in your ``$PATH``).
   This enables you to execute ``az`` when using the dockerized Azure CLI.
```

File: docs/azure/GettingStarted/AzureCLI.rst (hunk 1 above)
Comment (peknis) [inline docs/azure/GettingStarted/AzureCLI.rst]:
```suggestion
Create the az alias when using Docker
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

**Review by umapraseeda — APPROVED** [review]

**Pull request merged**